app/org/maproulette/models/dal/ChallengeDAL.scala

```diff
@@ -1041,7 +1054,11 @@ class ChallengeDAL @Inject() (
                                             WHEN t.priority = #${Challenge.PRIORITY_MEDIUM} THEN ${Challenge.PRIORITY_MEDIUM_NAME}
                                             WHEN t.priority = #${Challenge.PRIORITY_LOW} THEN ${Challenge.PRIORITY_LOW_NAME}
                                            END)) ||
-                                        hstore('mr_mappedOn', t.mapped_on::text) ||
+                                        hstore('mr_mappedOn',
```

Comment:
So I would imagine that the pg_timezone_names table is small, but this request will be made for every single returned row. How many records would this query generally be expected to return? You might want to look at the performance difference of doing a join, instead of a select for each returned record.

Reply:
Since the "select pg_timezone_names...." query isn't changing with each row the database optimizer performs that query once and saves the value. I verified this with an "explain" plan on a simplified version of the query.